### ISSUE TITLE
JSON API?

### ISSUE THREAD

**Issue by trevoristall** [mindbody/API-Examples#10]
I am building a Wordpress plugin for use with your API, having SOAP as a requirement makes it really hard for inexperienced people to setup and use. Is there any possibility of a JSON interface in the future?

**Comment by vacoj** [comment]
Unfortunately, SOAP/XML is all that you can use with this API.  As far as I know, there isn't a plan to place a JSON front-end onto it.

**Comment by ajitdsa** [comment]
What about using something like this? https://www.mashape.com/neurospeech/soap-to-json

**Comment by vacoj** [comment]
ajitdsa, 

I can't say that I approve of employing that service for the purposes of interacting with the Mindbody API.  One problem, among many, is that sensitive information could potentially be intercepted if ever there was a compromise.  

Additionally, we have no way of knowing how long requests are stored on the proxy's side.  According to the user agreement (found here: https://developers.mindbodyonline.com/Resources/DeveloperAgreement - section 8r): 

"You shall have no right to cache (in excess of 48 hours), collect, compile, store, transfer or utilize MINDBODY Data or any other data derived from MINDBODY, MINDBODY's computer system(s) or database(s), including but not limited to Cardholder Data, customer addresses, passwords or any other identifying information about any End User or End User Client."

In short, using a service like that could potentially open you, as the developer, up to risks that would not be present if you used direct communication with our servers via SOAP.
